app/vulncode_app/models.py

```diff
@@ -30,9 +29,11 @@ class Vulnerability(models.Model):
     A software vulnerability with minimal information.
     Identifiers are stored as VulnerabilityReference.
     """
-    summary = models.TextField(max_length=1024,
-                               help_text="Summary of the vulnerability")
-    cvss = models.FloatField(help_text="CVSS Score")
+    summary = models.CharField(max_length=50, help_text="Summary of the vulnerability", blank=True)
```

Comment:
Please use single quotes for all strings  (unless this is a docstring or  it contains a single quote itself)
This applies here and below